tests/utils/test_file_utils.py

```diff
@@ -81,25 +81,34 @@ def test_render_and_merge_yaml(tmpdir):
             test_1: [1, 2, 3]
             test_2: {{ TEST_VAR_1 }}
             test_3: {{ TEST_VAR_2 }}
+            test_4: {{ TEST_VAR_4 }}
             """
-            + r"test_4: {{{{ ('{0}' | from_json)['key'] }}}}".format(json_file)
+            + r"test_5: {{{{ ('{0}' | from_json)['key'] }}}}".format(json_file)
         )
-
-    data = {"MY_MLFLOW_SERVER": "./mlruns", "TEST_VAR_1": ["a", 1.2], "TEST_VAR_2": {"a": 2}}
     context_yaml_file = random_file("yaml")
-    file_utils.write_yaml(str(tmpdir), context_yaml_file, data)
+    with open(tmpdir / context_yaml_file, "w") as f:
```

Comment:
Here we additionally test using the `from_json` filter in the `context_yaml_file` (analogous to an MLflow `profile/*.yml` file)